pymc_marketing/clv/models/pareto_nbd.py

```diff
@@ -70,12 +69,21 @@ class ParetoNBDModel(CLVModel):
             * `recency`: time between the first and the last purchase
             * `T`: time between the first purchase and the end of the observation period; model assumptions require T >= recency
             * `customer_id`: unique customer identifier
+        Along with optional covariate columns.
+    pr_covar_columns: list, optional
+        List containing column names of covariates for customer purchase rates.
+    dr_covar_columns: list, optional
+        List containing column names of covariates for customer dropouts.
     model_config: dict, optional
         Dictionary containing model parameters:
-            * `r_prior`: Shape parameter of time between purchases for customer population; defaults to `pymc.Weibull.dist(alpha=2, beta=1)`
-            * `alpha_prior`: Scale parameter of time between purchases for customer population; defaults to `pymc.Weibull.dist(alpha=2, beta=10)`
-            * `s_prior`: Shape parameter of time until churn for customer population; defaults to `pymc.Weibull.dist(alpha=2, beta=1)`
-            * `beta_prior`: Scale parameter of time until churn for customer population; defaults to `pymc.Weibull.dist(alpha=2, beta=10)`
+            * `r_prior`: Shape parameter of time between purchases; defaults to `pymc.Weibull.dist(alpha=2, beta=1)`
+            * `alpha_prior`: Scale parameter of time between purchases; defaults to `pymc.Weibull.dist(alpha=2, beta=10)`
+            * `s_prior`: Shape parameter of time until dropout; defaults to `pymc.Weibull.dist(alpha=2, beta=1)`
+            * `beta_prior`: Scale parameter of time until dropout; defaults to `pymc.Weibull.dist(alpha=2, beta=10)`
+            * `alpha0_prior: Scale parameter of time between purchases if using covariates; defaults to `pymc.Weibull.dist(alpha=2, beta=10)`
```

Comment:
Why not reuse the kwarg `alpha_prior` since only one of them is used anyway depending on covariates?